app/models/concerns/has_wise_recipient.rb

```diff
@@ -0,0 +1,161 @@
+# frozen_string_literal: true
+
+module HasWiseRecipient
+  extend ActiveSupport::Concern
+
+  included do
+    include CountryEnumable
+    has_country_enum(field: :recipient_country)
+
+    # validate do
+    #   unless bic_code.nil? || bic_code.match(/[A-Z]{4}([A-Z]{2})[A-Z0-9]{2}([A-Z0-9]{3})?$/) # https://www.johndcook.com/blog/2024/01/29/swift/
+    #     errors.add(:bic_code, "is not a valid SWIFT / BIC code")
+    #   end
+    # end
+
+    # validate do
+    #   if IBAN_FORMATS[bank_country.to_sym] && !account_number.match(IBAN_FORMATS[bank_country.to_sym])
+    #     errors.add(:account_number, "does not meet the required format for this country")
+    #   end
+    # end
+
+    validate do
+      if POSTAL_CODE_FORMATS[recipient_country.to_sym] && !address_postal_code.match(POSTAL_CODE_FORMATS[recipient_country.to_sym])
+        errors.add(:address_postal_code, "does not meet the required format for this country")
+      end
+    end
```

Comment:
This was designed specifically for Column's required formats - not sure if Wise has different / looser requirements

Reply:
I have to imagine the postal code formats would be good to keep around? I don't see a world where Column has different format validations than the actual postal codes

Reply:
Sgtm - let's run with Column's for now